packages/macro-impact-api/tests/integration/endpoints.test.ts

```
@@ -0,0 +1,76 @@
+import { beforeEach, describe, expect, test } from "bun:test";
+import { buildApp } from "../../src/agent";
+
+const env = {
+  PAYMENTS_RECEIVABLE_ADDRESS: "0x1111111111111111111111111111111111111111",
+  FACILITATOR_URL: "https://facilitator.daydreams.systems",
+  NETWORK: "base",
+};
+
+describe("integration: macro impact API", () => {
+  beforeEach(() => {
+    Object.entries(env).forEach(([k, v]) => (process.env[k] = v));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)